Library/Homebrew/test/cask/cask_loader/from_api_loader_spec.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe Cask::CaskLoader::FromAPILoader, :cask do
-  shared_context "with API setup" do |new_token|
-    let(:token) { new_token }
-    let(:cask_from_source) { Cask::CaskLoader.load(token) }
+  shared_context "with API setup" do |local_token|
+    let(:api_token) { "#{local_token}-api" }
```

Comment:
Renamed here to prevent similar false positives in the future that caused the previous test suite to pass. By changing the token name we can be sure that it won't accidentally load a cask from a Ruby file on disk.